lib/install/validate/validator.go

```diff
@@ -361,6 +351,63 @@ func (v *Validator) managedbyVC(ctx context.Context) {
 	return
 }
 
+func (v *Validator) credentials(ctx context.Context, input *data.Data, conf *config.VirtualContainerHostConfigSpec) {
+	if input.OpsPassword == nil {
```

Comment:
Much as I hate it, an empty string is still a legitimate password. We've even run across environments where Bonneville wouldn't deploy because it was impossible to supply an empty password.